src/System.Private.CoreLib/Resources/Strings.resx

```diff
@@ -668,7 +668,7 @@
     <value>Insufficient memory to continue the execution of the program.</value>
   </data>
   <data name="Arg_ParamName_Name" xml:space="preserve">
-    <value>Parameter name: {0}</value>
+    <value>(parameter '{0}')</value>
```

Comment:
The typical error messages are sentences that end with a dot. I think it is why this error message started with upper case letter. It does not look right to me to start with lower case letter after a dot.